lib/galaxy/util/handlers.py

```diff
@@ -0,0 +1,132 @@
+"""Utilities for dealing with the Galaxy 'handler' process pattern.
+
+A 'handler' is a named Python process running the Galaxy application responsible
+for some activity such as queuing up jobs or scheduling workflows.
+"""
+
+import logging
+import os
+import random
+
+log = logging.getLogger( __name__ )
+
+
+class ConfiguresHandlers:
+
+    def _init_handlers(self, config_element):
+        # Parse handlers
+        if config_element is not None:
+            for handler in self._findall_with_required(config_element, 'handler'):
+                id = handler.get('id')
```

Comment:
This code was refactored out of jobs/__init__.py that had the same problem but I'll definitely change it - good catch.